.github/dependabot.yml

```diff
@@ -0,0 +1,16 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
+  # Enable version updates for GitHub Actions
+  - package-ecosystem: "github-actions"
+    # Workflow files stored in the default location of `.github/workflows`
+    # You don't need to specify `/.github/workflows` for `directory`. You can use `directory: "/"`.
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    commit-message:
+      prefix: ".github"
```

Comment:
Re-opening https://github.com/nix-community/nixvim/pull/3065#discussion_r1989552962:

I'd prefer this without the `.` personally. Ideally, we'd set it to whatever the most-used commit prefix for our workflows was historically (`ci`?), but either way I don't think we often included the `.` in commit messages?